writer.go

```diff
@@ -3,64 +3,189 @@ package log
 import (
 	"io"
 	"sync"
-	"time"
 )
 
+var MaxWriterBuffer = 16 * 1024 * 1024
```

Comment:
yeah... i was thinking like 512KB would be good, i cant imagine a single log message being larger than 1k.